python/activity_stream/widget_note.py

```diff
@@ -225,10 +225,10 @@ def set_note_info(self, data):
         self._set_timestamp(data, self.ui.date)
         
         # set the main note text
-        self.ui.content.setText(data["content"])
+        self.ui.content.setText(data.get("content", ""))
```

Comment:
I'll document it, but when we have a malformed Cut item, it's possible that we end up with a missing key here. It should be polished out above here, as well, but we need to make sure we don't have KeyErrors end up with the user.